docs/manual/src/udl/ext_types_external.md

```diff
@@ -29,9 +29,6 @@ dictionary ConsumingDict {
 
 ```
 
-(Note the special type `extern` used on the `typedef`. It is not currently enforced that the
-literal `extern` is used, but we hope to enforce this later, so please use it!)
-
```

Comment:
It seems like this wasn't referring to anything anymore, but please correct me if I missed the point.

Reply:
I think it is still relevant - you could change the example in the manual to `typedef foobar DemoDict;` and it would still work - that note is to suggest that we might in the future change things so it would not work without the explicit `extern` literal there.

(IIRC, the intent was that there might be a future where we that literal might mean something in some future, so we'd reserver `extern` for this behavior, but that seems unlikely)

Reply:
That makes sense, adding this back in.  I'm not sure where my brain was, I search and search and didn't see the `typedef` there.